public/src/utils/setupPaintProperties.js

```diff
@@ -23,6 +23,10 @@ export const setupPaintProperties = (p5, state) => {
   };
 };
 
+const scaleByZoomAmount = (coordinate, zoomAmount) => {
```

Comment:
accounts for the fact that the relationship between the raw mouse position and the (scaled/zoomed) canvas no longer has a 1:1 relationship, and just scales the values up based on the `zoomAmount`. these are also the same values being sent the server(!)